package.json

```diff
@@ -143,11 +143,11 @@
     "yaml-hook": "^1.0.0"
   },
   "optionalDependencies": {
-    "@malept/electron-installer-flatpak": "^0.11.2",
```

Comment:
these aren't needed for this PR- let me know if it would be more preferred to separate these upgrades to a separate PR

Reply:
I'm fine with upgrading these in this PR, as long as tests are passing 👍 We can do a little manual test of binaries next time we do a Forge release, but the upgrade should be safe